Cython/Compiler/ExprNodes.py

```diff
@@ -10691,6 +10699,16 @@ def calculate_result_code(self):
 class DecrementIncrementNode(CUnopNode):
     #  unary ++/-- operator
 
+    def type_error(self):
```

Comment:
```suggestion
    is_inc_dec_op = True

    def type_error(self):
```

The final part of this change is that `UnopNode` needs to have `is_inc_dec_op = False` but I can't suggest that through the github interface